Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -0,0 +1,1653 @@
+            'Name': demisto.get(alert, 'policy.name'),
+            'Type': demisto.get(alert, 'policy.policyType'),
+            'Severity': demisto.get(alert, 'policy.severity'),
+            'Remediable': demisto.get(alert, 'policy.remediable')
+        },
+        'Resource': {
+            'ID': demisto.get(alert, 'resource.id'),
+            'Name': demisto.get(alert, 'resource.name'),
+            'Account': demisto.get(alert, 'resource.account'),
+            'AccountID': demisto.get(alert, 'resource.accountId')
+        }
+    }
+    if 'resource_keys' in args:
+        # if resource_keys argument was given, include those items from resource.data
+        extra_keys = demisto.getArg('resource_keys')
+        resource_data = {}
+        keys = extra_keys.split(',')
+        for key in keys:
+            resource_data[key] = demisto.get(alert, f'resource.data.{key}')
+
+        ec['Resource']['Data'] = resource_data
+
+    if alert.get('alertRules'):
+        ec['AlertRules'] = [alert_rule.get('name') for alert_rule in alert.get('alertRules')]
+
+    return ec
+
+
+def format_v1_response(response: Any) -> Any:
+    """
+    This was copied from V1 and should not be maintained.
+    """
+    if response and isinstance(response, dict):
+        response = {pascalToSpace(key).replace(" ", ""): format_v1_response(value) for key, value in response.items()}
+    elif response and isinstance(response, list):
+        response = [format_v1_response(item) for item in response]
+    return response
+
+
+def alert_search_v1_command(client: Client, args: Dict[str, Any], return_v1_output: bool) -> \
+        Union[CommandResults, List[Union[CommandResults, str]], Dict]:
+    new_args = {
+        'time_range_unit': args.get('time-range-unit'),
+        'time_range_value': args.get('time-range-value'),
+        'time_range_date_from': args.get('time-range-date-from'),
+        'time_range_date_to': args.get('time-range-date-to'),
+        'filters': get_v1_filters(args),
+        'limit': args.get('limit', DEFAULT_LIMIT),
+        'detailed': 'true'
+    }
+
+    command_results = alert_search_command(client, new_args)
+    if return_v1_output:
+        response = command_results.raw_response
+
+        context_path = 'Redlock.Alert(val.ID === obj.ID)'
+        context: dict = {context_path: []}
+        for alert in response:  # type: ignore[attr-defined]
+            context[context_path].append(alert_to_v1_context(alert, args))
+        context['Redlock.Metadata.CountOfAlerts'] = len(response)  # type: ignore[arg-type]
+
+        command_results = command_results.to_context()
+        command_results['EntryContext'].update(context)  # type: ignore[index]
+
+    if args.get('risk-grade'):
+        return [RISK_GRADE_NOT_SUPPORTED_MSG, command_results]
+
+    return command_results
+
+
+def alert_get_details_v1_command(client: Client, args: Dict[str, Any], return_v1_output: bool) -> Union[CommandResults, Dict]:
+    new_args = {
```

Comment:
please add a comment in all v1 methods, explaining why they're here and not standard, for future maintainers